src/components/common/YourCompany/YourCompany.vue

```diff
@@ -590,7 +590,7 @@ export default class YourCompany extends Mixins(
     color: $app-red !important;
   }
   #contact-info-edit-btn {
-    padding-right: 4px;
+    margin-right: -14px;
```

Comment:
This "pulls" the button to the right, so the button label lines up with the right margin, and the button keeps its padding (which goes past the margin). Example:

![image](https://user-images.githubusercontent.com/10002889/170584241-23f734f1-9f57-47cf-b52b-dc99d54a488d.png)